tests/linearizability/model.go

```diff
@@ -151,3 +164,15 @@ func stepPut(state EtcdState, request etcdRequest, response etcdResponse) (bool,
 	state.LastRevision = response.revision
 	return true, state
 }
+
+func stepDelete(state EtcdState, request etcdRequest, response etcdResponse) (bool, EtcdState) {
+	if response.err != nil {
+		state.FailedWrites[request.putData] = struct{}{}
```

Comment:
```suggestion
		state.FailedWrites[""] = struct{}{}
```
It's true that `request.putData` for delete request will be empty, but let's not use fields meant only for put requests.